packages/@ember/-internals/glimmer/lib/component.ts

```diff
@@ -655,7 +655,7 @@ const Component = CoreView.extend(
      */
     readDOMAttr(name: string) {
       // TODO revisit this
```

Comment:
assuming this comment is related to the type inference(`as HTMLElement`), seems it can be removed now